### PR TITLE
[stable/sentry] Fix inconsistent SENTRY_SECRET_KEY among the pods when not set explicitly.

### PR DESCRIPTION
**Pull request by Antiarchitect**
#### What this PR does / why we need it:
Make Deployment and Job pods dependent on `secret.yaml` content (especially SENTRY_SECRET_KEY when it is not set explicitly) follow the Secret changes in sync. Symptoms were (after some deployments cannot log in, after some deployments cannot accept errors). Also, document the ability to set SENTRY_SECRET_KEY explicitly.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ x ] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [ x ] Chart Version bumped
- [ x ] Variables are documented in the README.md
- [ x ] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/sentry/README.md [hunk 1]
```diff
@@ -66,6 +66,7 @@ Parameter                                            | Description
 `image.tag`                                          | Sentry image tag                                                                                           | `9.1.1`
 `imagePullPolicy`                                    | Image pull policy                                                                                          | `IfNotPresent`
 `imagePullSecrets`                                   | Specify image pull secrets                                                                                 | `[]`
+`sentrySecret`                                       | Specify fixed SENTRY_SECRET_KEY                                                                            | `nil`
```
Comment (okgolove) [inline stable/sentry/README.md]: I would change this.
Something like:
`Specify SENTRY_SECRET_KEY. If isn't specified it will be generated automatically.`

Comment (Antiarchitect) [reply stable/sentry/README.md]: Done

**Review by okgolove — APPROVED** [review]

**Pull request merged**